lib/profiler/mitmDumpProfiler.py

```diff
@@ -0,0 +1,38 @@
+import os
+import platform
+import subprocess
+from base import BaseProfiler
+
+
+class MitmDumpProfiler(BaseProfiler):
+
+    process = None
+
+    def start_recording(self):
+        default_dump_flow_cmd = ["mitmdump", "--no-http2", "-w"]
+        flow_file_fp = os.path.join(self.env.DEFAULT_FLOW_DIR, self.env.flow_file_fp)
+        if not os.path.exists(self.env.DEFAULT_FLOW_DIR):
+            os.mkdir(self.env.DEFAULT_FLOW_DIR)
```

Comment:
just like what we discuss before, I'll suggest using `mkdir -p`-liked method.
And now, I found it, `os.makedirs()`, from https://docs.python.org/2/library/os.html#os.makedirs :P